src/inmanta/data/schema.py

```diff
@@ -0,0 +1,211 @@
+import logging
+import pkgutil
+from types import ModuleType
+from typing import List, Optional
+
+import asyncpg
+from asyncpg import UndefinedTableError
+
+LOGGER = logging.getLogger(__name__)
+
+LEGACY_SCHEMA_VERSION_TABLE = "schemaversion"
+SCHEMA_VERSION_TABLE = "schemamanager"
+
+# no version table found
+VERSION_NONE = None
+# Legacy version table is present but empty (unlikely)
+VERSION_LEGACY = 0
+
+CORE_NAME = "core"
```

Comment:
What is the difference between CORE_NAME and CORE_DATABASE_NAME in data/__init__.py?